main.go

```diff
@@ -134,9 +147,17 @@ func buildClusterDetails(svc kafkaClient, c types.ClusterInfo) (clusterDetails,
 
 func filterClusters(clusters kafka.ListClustersOutput, filter regexp.Regexp) *kafka.ListClustersOutput {
```

Comment:
Now that we're adding additional ways to filter we might want to pass in a more generic `Filter` type which contains the cluster name filter and any tag filters rather than passing the name filter in as an argument but then using the global `tagFilters` for tags. It'd be easier to test that way as well without modifying global vars.